apps/src/applab/sanitizeHtml.js

```diff
@@ -200,7 +200,15 @@ export default function sanitizeHtml(
       'placeholder',
       'step',
       'type',
-      'value'
+      'value',
+      'accept',
+      'hidden',
+      'capture'
+    ]),
+    label: standardAttributes.concat([
+      'data-canonical-image-url',
```

Comment:
can we remove this attribute (and/or any of the other new attributes) now that the camera icon uses font-awesome?